src/commands/convert.ts

```diff
@@ -82,8 +82,7 @@ export default class Convert extends Command {
         metadata['success'] = true;
         metadata['from_version'] = document.version();
         metadata['to_version'] = flags['target-version'];
-        console.log(metadata);
```

Comment:
Tests were failing due to this console.log printing things to stdout. Some of our integration tests are based on the stdout made by the cli and this was polluting it.